modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,76 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying virtual router conflicts
+
+This procedure is only required if you have multiple clusters using the same network.
+
+The Openshift cluster Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router-id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have multiple clusters using the same network, you should check to see which VIPs the installation will choose to avoid conflicts.
```

Comment:
Done.

Reply:
Preview link to this file: https://deploy-preview-35316--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-default.html#keepalived-limitation-for-ipi_installing-rhv-default

- Is there a reason why you want to use lowercase `id` in the general form? When referring to a literal value, we generally wrap in backticks and follow with the type, i.e., `virtual-router-id` number. 

-  "Duplication of the ids..." - "ids" also looks odd in lowercase here.

- "The Openshift cluster Virtual IPs" should include the initialism `(VIPs)` in the first instance here  since you refer to "VIP" subsequently in the procedure.